api/db/changelog/db.changelog-29-admin-logging.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="rsasch" id="changelog-29">
+    <createTable tableName="admin_action_history">
+      <column name="history_id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="user_id" type="bigint">
```

Comment:
opt: admin_user_id might be clearer

Reply:
Since we might make a distinction in the database between admin and non-admin users somewhere in the future, I'd prefer to wait until we sort that out first.

Reply:
The reason that I find this confusing is that there may be more than one user involved in this record, e.g. the admin user and a user targeted by the action, therefore just "user_id" could be ambiguous. The table is also called `admin_action_history` so it seems like you're pretty committed to the source user being an admin.

Unless I'm misunderstanding what this column represents.

Reply:
good point